functions/java-jre.bash

```diff
@@ -1,4 +1,21 @@
 #!/usr/bin/env bash
+# shellcheck disable=SC2181
+
+java_install(){
+  if is_x86_64; then
+    if ! [ -x "$(command -v java)" ] || [[ ! "$(java -version)" == *"Zulu"* ]]; then
+      cond_redirect systemctl stop openhab2.service
+      java_zulu_enterprise_8_apt
+      cond_redirect systemctl start openhab2.service
+    fi
+  elif [ java_zulu_tar_update_available ]; then
```

Comment:
new spellcheck warning, maybe [] can be removed?

Reply:
Aha, missed to run shellcheck on last commit. Will do another round with that. 👍